runagent/utils/agent.py

```
@@ -218,7 +234,7 @@ def validate_pythonic_agent(config, dynamic_loading, folder_path):
 
     # Detect framework
     try:
-        validation_details["framework"] = detect_framework(folder_path)
+        validation_details["framework"] = detect_framework(folder_path).value
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)